person.rb

```diff
@@ -20,4 +20,8 @@ def of_age?()
   def can_use_service?
     of_age? || @parents_permissiom
```

Comment:
- [ ] **[OPTIONAL]** What you have in the Person `initialize` method is `parents_permission`, not `parents_permissiom` (with 'm'). I would recommend you fix that.